src/modules/home/home.user/UserHomeView.tsx

```
@@ -1,11 +1,14 @@
 import { Button } from '@/components/ui/button'
 import { PATHS } from '@/config'
+import { useGetAllProducts } from '@/queries/products'
 import { useNavigate } from 'react-router-dom'
 
 type Props = {}
 
 const UserHomeContainer = ({}: Props) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)